test/access/Roles.test.js

```diff
@@ -14,16 +15,16 @@ contract('Roles', function ([_, authorized, otherAuthorized, other]) {
 
   context('initially', function () {
     it('doesn\'t pre-assign roles', async function () {
-      (await this.roles.has(authorized)).should.equal(false);
-      (await this.roles.has(otherAuthorized)).should.equal(false);
-      (await this.roles.has(other)).should.equal(false);
+      expect( await this.roles.has(authorized)).to.equal(false);
```

Comment:
The linter (`npm run lint:js`) is failing because of this extra space, I think we can remove it without hindering readability.
```suggestion
      expect(await this.roles.has(authorized)).to.equal(false);
```